server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/templates_config_representer.rb

```diff
@@ -0,0 +1,28 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Config
+    class TemplatesConfigRepresenter < ApiV1::BaseRepresenter
```

Comment:
No links like in [here](https://github.com/gocd/gocd/blob/f18d7930b9213a5a46b37ffc1c24b9365fd15ac9/server/webapp/WEB-INF/rails.new/app/presenters/api_v4/agents_representer.rb#L26-L32)?